File: java/OOAD/TicTacToe/src/com/techlabs/tictactoe/Player.java
package com.techlabs.tictactoe;

public class Player {
	private String name;
	private Mark mark;
	
	public Player(String name,Mark mark){
		this.name=name;
		this.mark=mark;
	}
	public Mark getMark(){
		return this.mark;
	}
	public String getName(){
		return this.name;
	}
}

File: dotNet/classlibrary/WinForm/TicTacToeApp/TicTacToeCore/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TicTacToeCore
{
   public  class Player
    {
        private string _name;
        private MarkType _mark;

        public Player(string name,MarkType mark)
        {
            this._name = name;
            this._mark = mark;
        }

        public MarkType Mark
        {
            get
            {
                return _mark;
            }
        }

        public string Name
        {
            get
            {
                return this._name;
            }
        }

    }
}
